.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
no.

I will disable the in game system and order admins to refuse to create bug reports if you force people to use bullshit walk-through question style reporting forms.

Reply:
Hammer unneeded but point is fully valid, people should have the option to ignore the template.

Reply:
alright that's why i asked what your thoughts were